tests/libres_tests/res/enkf/test_enkf_load_results_manually.py

```diff
@@ -70,3 +72,39 @@ def test_load_results_from_run_context(self):
             self.assertEqual(24, loaded)
             self.assertEqual(25, len(expected))
             self.assertEqual(25, len(realisations))
+
+
+class LoadResultsManuallyLogtest(ResTest):
+    """
+    This little test does not depend on Equinor-data and only verifies
+    the lazy_load flag in forward_load_context plus memory-logging
+    """
+
+    def setUp(self):
+        self.config_file = self.createTestPath("local/snake_oil/snake_oil.ert")
+
+    @pytest.fixture(autouse=True)
+    def setup_fixtures(self, caplog):
+        self._caplog = caplog
+
+    @tmpdir()
+    def test_load_results_manually(self):
```

Comment:
Would suggest doing this in `pytest` where there is native support for `caplog` and monkeypatching of environment variables.

Reply:
Could potentially use this: https://github.com/equinor/ert/blob/9ff386bf346dd3ccedee213886a3420b38f574c6/tests/libres_tests/res/enkf/test_es_update.py#L13

Reply:
Ahh...  brilliant! Didn't know about that one - thanks.